### PR TITLE
group test updates

### PR DESCRIPTION
**Pull request by mpeck**
part of #1122

### CODE REVIEW

File: test/commands/group_test.exs [hunk 1]
```diff
@@ -1,131 +1,155 @@
...
 
-  test "listing group", %{user: user} do
+  test "listing group" do
     group("elves")
     cheer = group("cheer")
+    user = user("belf")
 
-    [response] = send_message(user, "@bot: operable:group member add cheer belf")
+    {:ok, response} = new_req(args: ["member", "add", "cheer", "belf"])
+    |> send_req()
```
Comment (christophermaier) [inline test/commands/group_test.exs]: Similarly, this is setup code. The assertions that follow are unnecessary.

**Comment by mpeck** [comment]
I can rework this guy. I started doing that on some of the previous modules, but decided to just convert everything over as it is because it was taking a good bit of time. But it's probably not a bad idea to go ahead and make those improvements now.

File: test/commands/group_test.exs [hunk 2]
```diff
@@ -1,176 +1,245 @@
...
 
-    [response] = send_message(user, "@bot: operable:group role add cheer admin")
-    assert response.name == "cheer"
+      {:ok, response} =
+        new_req(args: ["rename", "renamed_group", "has_been_renamed"])
+        |> send_req()
 
-    [response] = send_message(user, "@bot: operable:group info cheer")
-    assert hd(response.roles).name == "admin"
+      assert(%{name: "has_been_renamed"} = response)
```
Comment (kevsmith) [inline test/commands/group_test.exs]: I think you want to use `==` here, no?

File: test/commands/group_test.exs [hunk 3]
```diff
@@ -1,176 +1,245 @@
...
-    [response] = send_message(user, "@bot: operable:group info cheer")
-    assert length(response.roles) == 0
-  end
+      {:ok, response} =
+        new_req(args: ["info", "info_group"])
+        |> send_req()
 
-  test "getting group info", %{user: user} do
-    group("cheer")
+      assert(%{name: "info_group"} = response)
```
Comment (kevsmith) [inline test/commands/group_test.exs]: Same here re: `==`

File: test/commands/group_test.exs [hunk 4]
```diff
@@ -1,176 +1,245 @@
...
-    assert_error_message_contains(response, "name: has already been taken")
-  end
+      response = Enum.sort_by(response, &(&1.name))
 
-  test "errors using the group command", %{user: user} do
-    [response] = send_message(user, "@bot: operable:group create")
-    assert response.error == "Missing required argument: group_name"
+      assert([%{name: "group1"},
+              %{name: "group2"},
+              %{name: "group3"}] = response)
```
Comment (kevsmith) [inline test/commands/group_test.exs]: Same here re: `==`

File: test/commands/group_test.exs (hunk 2 above)
Comment (mpeck) [inline test/commands/group_test.exs]: Nope `=` is correct. There is actually more to the record than just the name. There is also an id, which we don't care about, and `members` and `roles` keys that are empty.

Actually as I think about it, I should have included the members and roles to confirm that they are empty.

Comment (mpeck) [inline test/commands/group_test.exs]: The more I look at it, I think it would actually be clearer to `match?/2` instead.

**Pull request merged**